src/main/java/seedu/hirehub/logic/CommandBoxState.java

```diff
@@ -6,6 +6,7 @@
  */
 public enum CommandBoxState {
     NORMAL,
+    CONFIRM,
     CLEARCONFIRM,
     DELETECONFIRM,
     DELETEAPPLICATIONCONFIRM,
```

Comment:
what is the purpose of these three enum states? CMIIW, but I don't see them used anywhere after the refactoring

Reply:
forgot to remove them, fixed